.github/workflows/weekly-acceptance-1-2-x.yml

```diff
@@ -0,0 +1,31 @@
+# Dispatch to the consul-k8s-workflows with a weekly cron
+#
+# A separate file is needed for each release because the cron schedules are different for each release.
+name: weekly-acceptance-1-2-x
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    # Run weekly on Wednesday at 3AM UTC/11PM EST/8PM PST
+   # - cron:  '0 3 * * 3'
+    - cron:  '0 0 * * *' # Temporarily nightly until 1.2.0 GA
+
+
+# these should be the only settings that you will ever need to change
+env:
+  CONSUL_IMAGE: hashicorppreview/consul-enterprise:1.16-dev # Consul's enterprise version to use in tests
```

Comment:
This is debt from other changes but you do not need CONSUL_IMAGE. Same goes for the inputs on the last line.